src/omotes_simulator_core/solver/network/assets/solver_pipe.py

```diff
@@ -493,79 +444,34 @@ def _calculate_total_heat_transfer_coefficient(
         else:
             return self.alpha_value
 
-    def _internal_energy_steady_state_objective(
-        self,
-        internal_energy_iteration: float,
-        internal_energy: float,
-        mass_flow_rate: float,
-    ) -> float:
-        r"""Root function for the internal energy steady state equation.
-
-        Solves the following optimization objective:
-
-        .. math::
-
-            \dot{m} \left( e_{i-1} - e_{i} \right) - \alpha A \left( T_{i} - T_{\text{ambient}}
-            \right) = 0
-
-        :param float internal_energy_iteration: The internal energy at the current grid point.
-        :param float internal_energy: The internal energy at the previous grid point.
-        :param float mass_flow_rate: The mass flow rate of the fluid.
-
-        """
-        # Calculate the temperature of the fluid
-        temperature = fluid_props.get_t(internal_energy_iteration)
-        # Calculate the total heat transfer coefficient
-        total_heat_transfer_coefficient = self._calculate_total_heat_transfer_coefficient(
-            temperature=temperature, mass_flow_rate=mass_flow_rate
-        )
-        # Return the objective function
-        element_size = self.length / self._grid_size
-        objective = mass_flow_rate * (
-            internal_energy - internal_energy_iteration
-        ) - total_heat_transfer_coefficient * np.pi * self.diameter * element_size * (
-            temperature - self.ambient_temperature
-        )
-        return float(np.array(objective).item())
-
-    def _update_internal_energy_grid(self) -> float:
-        r"""Update the internal energy grid of the pipe.
-
-        Iteratively calculate the internal energy at each grid point of the pipe.
+    def _calculate_total_heat_loss(self, tin: float, mass_flow_rate: float) -> float:
+        r"""Calculate the total heat loss of the pipe.
 
-        Solve the following equation for the internal energy at each grid point:
+        The heat loss is calculated with the following formula:
         .. math::
 
-            \dot{m} \left( e_{i-1} - e_{i} \right) = \alpha A \left( T_{i} -
-            T_{\text{ambient}} \right)
+        \dot{m} c_p (T_{in} - T_{out}) (1 - \exp(-\alpha \pi D L
```

Comment:
Use \frac{}{} to define a fraction, and \left( \right) to create brackets around T_in and T_out.

Reply:
fixed